camel-chimera-graph/src/main/java/com/cefriel/component/GraphConsumer.java

```diff
@@ -44,8 +44,21 @@ protected void doStart() throws Exception {
         } else {
             operationConfig = new GraphBean();
         }
+
+        // todo name of operation can either come from endpoint.getName() or endpoint.getOperationName()
         operationConfig.setEndpointParameters(endpoint);
-        if ("get".equals(endpoint.getName())) {
+
+        String operation;
+        if (endpoint.getName().equals("") && endpoint.getOperation() != null)
+            operation = endpoint.getOperation();
+        else if (endpoint.getName() != null && endpoint.getOperation() == null)
```

Comment:
Check w.r.t. the same condition for inputFormat parameter in mapt component. What should be done if both are provided?